tests/queue_counter_test.py

```diff
@@ -1262,6 +1311,25 @@ def test_queue_voq_counters_port_json(self):
             del v["time"]
         assert json_dump(json_output) == show_queue_port_voq_counters_json
 
+    @pytest.mark.parametrize("counter,counter_type,output", [
```

Comment:
This test does not belong here. it should go to 
https://github.com/sonic-net/sonic-utilities/blob/master/tests/watermarkstat_test.py